static/app/views/codecov/coverage/commits/commitDetailSummary.tsx

```diff
@@ -0,0 +1,130 @@
+import styled from '@emotion/styled';
+
+import {
+  SummaryContainer,
+  SummaryEntries,
+  SummaryEntry,
+  SummaryEntryLabel,
+  SummaryEntryValue,
+  SummaryEntryValueLink,
+} from 'sentry/components/codecov/summary';
+import Link from 'sentry/components/links/link';
+import Panel from 'sentry/components/panels/panel';
+import PanelBody from 'sentry/components/panels/panelBody';
+import PanelHeader from 'sentry/components/panels/panelHeader';
+
+const headCommit = {
+  sha: '31b72ff64bd75326ea5e43bf8e93b415db56cb62',
+  shortSha: '31b72ff',
+};
+
+const baseCommit = {
+  sha: 'da46d4c13e4a75b7624c8c6763816ecb6dad1968',
+  shortSha: 'da46d4c',
+};
+
+export function CommitDetailSummary() {
+  return (
+    <SummaryContainer columns={12}>
+      <SelectedCommitPanel>
+        <PanelHeader>Coverage On Selected Commit</PanelHeader>
+        <PanelBody>
+          <SummaryEntries largeColumnSpan={6} smallColumnSpan={2}>
+            <SummaryEntry>
+              <SummaryEntryLabel showUnderline body={<p>Repository coverage tooltip</p>}>
+                Repository coverage
```

Comment:
Do we need the t('...') locale library for all of these?

Reply:
Ahhh yea you're right 😭